dll/win32/browseui/lang/fr-FR.rc

```diff
@@ -354,3 +354,9 @@ BEGIN
     IDS_COL_LOCATION "Dans le dossier"
     IDS_COL_RELEVANCE "Pertinence"
 END
+
+STRINGTABLE
+BEGIN
+    IDS_PARSE_ADDR_ERR_TITLE "File Explorer"
+    IDS_PARSE_ADDR_ERR_TEXT "File Explorer can't find '%1'. Check the spelling and try again."
```

Comment:
```suggestion
    IDS_PARSE_ADDR_ERR_TEXT "L'explorateur de fichiers ne peut trouver '%1'. Veuillez vérifier l'orthographe et réessayer."
```

Reply:
Also in english, "can't" doesn't look too much professional. "cannot" would be better.

Reply:
Fair enough - I took similar verbiage from win10, but xp does use "cannot".

Reply:
Windows <= 2003 use the "pro" verbiage; vista+ use the "chad" version LMAO :sweat_smile: :sweat_smile:

Reply:
Went for 'pro'.